### DISCUSSION TITLE
PFN mapping request form is unavailable

### DISCUSSION THREAD

**Discussion by ice02** [microsoft/WindowsAppSDK#2176] [Q&A] [answered]
Hello,
I try to experimente push notifications with Windows App SDK but the form (https://aka.ms/windowsappsdk-push-mapping) to ask for PFN mapping seems off since few weeks now.
Is the procedure changed or will it be online again soon?
Thanks a lot by advance for the answer

**Comment by nicuparente** [comment, marked as answer]
Hi @ice02 - we are finalizing the new process for PFN mapping, and will be publishing the new docs soon. We appreciate your patience.

**Reply by ice02** [reply]
thanks for the quick answer
